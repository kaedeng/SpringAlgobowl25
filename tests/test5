10 10
2 3 1 2 2 3 2 3 2 1 
2 2 3 2 3 3 2 3 0 2 
..T.......
....T.T...
.T.TT.T...
..........
...T...TT.
.T..TT.T..
T..T..T...
.....T....
T.TT.T....
......T...
