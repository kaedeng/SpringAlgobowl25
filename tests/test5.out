38
15
1 1 X
5 10 L
4 6 X
10 8 L
2 4 R
3 3 R
6 4 D
8 3 D
3 6 L
7 8 U
5 7 R
8 7 U
8 5 R
1 7 D
7 6 U